examples/ensemble/plot_adaboost_regression.py

```diff
@@ -9,26 +9,38 @@
 regressor. As the number of boosts is increased the regressor can fit more
 detail.
 
-.. [1] H. Drucker, "Improving Regressors using Boosting Techniques", 1997.
+.. [1] H Drucker, "Improving Regressors using Boosting Techniques", 1997.
```

Comment:
Do you mind adding the link to the proceeding?
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.21.5683&rep=rep1&type=pdf
I could not find the DOI (perhaps the article is still too old)... if by any chance you can find it, you might want to use the :doi: directive for the link.